sys/amd64/start.S

```diff
@@ -0,0 +1,46 @@
+#include <amd64/asm.h>
+#define ASM_FILE
```

Comment:
I would like to defer employing genasym to some future PR. For now, I'm not so sure what to put inside `genasym.cf` in case of AMD64.

Reply:
You do exactly the same thing which genasym does - preprocessing c file in order to make it valid for asm. It is a place where you __should__ use genasym to avoid magic `ASM_FILE` definition. So please don't reinvent something that works.

Reply:
Done.